src/app/components/pool-size-chart/pool-size-chart.component.ts

```diff
@@ -31,10 +31,66 @@ export class PoolSizeChartComponent implements OnInit {
   ) {}
 
   ngOnInit(): void {
-    this.setupChart();
+    this.config = {
+      curveType: 'function',
+      legend: 'none',
+      backgroundColor: 'transparent',
+      colors: [this.palette.series],
+      chartArea: {
+        width: '90%',
+        height: '80%',
+        right: 0,
+        left: 90,
+      },
+      hAxis: {
+        viewWindowMode: 'maximized',
+        textStyle: {
+          color: this.palette.text,
+        },
+        gridlines: {
+          color: this.palette.gridlines,
+          count: 1,
+        },
+        minorGridlines: {
+          color: this.palette.gridlines,
+          count: 0,
+        },
+      },
+      vAxis: {
+        logScale: false,
+        viewWindowMode: 'maximized',
+        format: '#.#TB',
+        textStyle: {
+          color: this.palette.text,
+        },
+        gridlines: {
+          color: this.palette.gridlines,
+        },
+        minorGridlines: {
+          count: 0,
+        }
+      },
+      pointSize: 3,
+    };
+    
+    this.setupChart(this.config);
+  }
+
+  ngOnChanges(changes : any) {
+    if ( changes.config != undefined ) {
```

Comment:
Looks like linter didn't work for you. Files were supposed to get linted before commit, can you confirm that something's not working here? 
Also please do `npm run lint:fix`